src/Raven.Server/ServerWide/ClusterCommandsVersionManager.cs

```diff
@@ -106,7 +106,8 @@ public static class ClusterCommandsVersionManager
             [nameof(EditDatabaseClientConfigurationCommand)] = Base42CommandsVersion,
             [nameof(PutServerWideBackupConfigurationCommand)] = 42_001,
             [nameof(DeleteServerWideBackupConfigurationCommand)] = 42_001,
-            [nameof(UpdateUnusedDatabaseIdsCommand)] = 42_002
+            [nameof(UpdateUnusedDatabaseIdsCommand)] = 42_002,
+            [nameof(UpdateLicenseLimitsCommand)] = 42_002,
```

Comment:
This needs to be 42_003? @karmeli87 ?

Reply:
UpdateUnusedDatabaseIdsCommand isn't in the stable yet

Reply:
Ok